app/components/radios/nested-conditionals.njk

```diff
@@ -0,0 +1,128 @@
+{% set html_style = 'background-color: #f0f4f5;' %}
+{% set title = 'Radios with nested conditonal content' %}
+{% from 'components/radios/macro.njk' import radios %}
+{% from 'components/input/macro.njk' import input %}
+{% from 'components/warning-callout/macro.njk' import warningCallout %}
+{% extends 'layout.njk' %}
+
+{% set emailHtml %}
+  {{ input({
+    id: "email",
+    name: "email",
+    classes: "nhsuk-u-width-two-thirds",
+    label: {
+      text: "Email address"
+    }
+  }) }}
+{% endset -%}
+
+{% set phoneHtml %}
+  {{ input({
+    id: "phone",
+    name: "phone",
+    classes: "nhsuk-u-width-two-thirds",
+    label: {
+      text: "Phone number"
+    }
+  }) }}
+{% endset -%}
+
+{% set mobileHtml %}
+  {{ input({
+    id: "mobile",
+    name: "mobile",
+    classes: "nhsuk-u-width-two-thirds",
+    label: {
+      text: "Mobile phone number"
+    }
+  }) }}
+{% endset -%}
+
+{%- set nestedConditional %}
+    {{ radios({
+      "idPrefix": "inner",
+      "name": "inner",
+      "fieldset": {
+        "legend": {
+          "text": "How would you prefer to be contacted?",
+          "classes": "nhsuk-fieldset__legend--l",
+          "isPageHeading": "true"
```

Comment:
Should we remove this `isPageHeading` to prevent there being multiple `<h1>` on the page?